receiver/nginxreceiver/scraper_test.go

```diff
@@ -133,8 +138,273 @@ func TestScraperError(t *testing.T) {
 	})
 }
 
+func TestScraperLogs(t *testing.T) {
+	l, err := net.Listen("tcp", "127.0.0.1:52418")
+	require.Nil(t, err)
```

Comment:
Use `require.NoError(t, err)` when asserting there should be no error.

Reply:
fixed